README.md

```diff
@@ -68,6 +72,10 @@ Add a custom card or custom element in your `ui-lovelace.yaml` using `type: cust
   stroke_width: 10
   font_size: 1.5em
   gradient: true
+  units: ' '
+  attribute: 'ambient'
+  font_shadow: '2px 2px black'
+  font-family: 'Trebuchet MS'
```

Comment:
I don't like having a separate config value for each style. This could get out of hand easily. This should be implemented using a `style` config similar to lovelace `picture-elements` that allows people to specify any css attribute they want: https://github.com/home-assistant/home-assistant-polymer/blob/db310646b7563178a4d30f5f5a1433186638f509/src/panels/lovelace/cards/hui-picture-elements-card.js#L88